packages/microcosm-preact/src/presenter.js

```diff
@@ -139,13 +138,11 @@ class PresenterMediator extends Component {
 
     this.presenter = props.presenter
     this.repo = this.presenter._requestRepo(context.repo)
-    this._lastRevision = -Infinity
 
     this.model = new Model(this.repo, this.presenter)
 
     // The following methods are autobound to protect scope
     this.send = this.send.bind(this)
-    this._scheduleUpdate = this._scheduleUpdate.bind(this)
```

Comment:
Similarly, Presenters don't deal with scheduling any more. This is all handled in the Model.